docs/blocks/blocks-16.md

```diff
@@ -585,113 +633,130 @@ See [this](/blocks/block-tags) page for a list of vanilla tags and relavent bloc
 
 ### minecraft:crafting_table
 
+Turns your block into a new/custom Crafting Table.
+
 <CodeHeader></CodeHeader>
 
 ```json
 {
 	"minecraft:crafting_table": {
 		"custom_description": "Example Crafting Table", // Name shown in GUI
 		"grid_size": 3, // Currently only supports 3
-		"crafting_tags": ["crafting_table", "custom_crafting_tag"]
+		"crafting_tags": ["crafting_table", "custom_crafting_tag"] //Tags to be used within the recipe file
 	}
 }
 ```
 
 ### minecraft:on_step_on
 
+Runs an event when an entity stepped on the block.
+
 <CodeHeader></CodeHeader>
 
 ```json
 {
 	"minecraft:on_step_on": {
 		"event": "block_event",
 		"target": "self",
-		"condition": "query.block_property('bedrock_wiki:block_property') == true"
+		"condition": "query.block_property('bedrock_wiki:block_property') == true" //Optional
 	}
 }
 ```
 
 ### minecraft:on_step_off
 
+Runs an event when an entity stepped off of the block.
+
 <CodeHeader></CodeHeader>
 
 ```json
 {
 	"minecraft:on_step_off": {
 		"event": "block_event",
 		"target": "self",
-		"condition": "query.block_property('bedrock_wiki:block_property') == true"
+		"condition": "query.block_property('bedrock_wiki:block_property') == true" //Optional
 	}
 }
 ```
 
 ### minecraft:on_fall_on
 
+Runs an event when an entity fell on the block.
+
 <CodeHeader></CodeHeader>
 
 ```json
 {
 	"minecraft:on_fall_on": {
 		"event": "block_event",
 		"target": "self",
-		"condition": "query.block_property('bedrock_wiki:block_property') == true",
+		"condition": "query.block_property('bedrock_wiki:block_property') == true", //Optional
 		"min_fall_distance": 5
 	}
 }
 ```
 
 ### minecraft:on_placed
 
+Runs an event when an entity placed the block.
+
 <CodeHeader></CodeHeader>
 
 ```json
 {
 	"minecraft:on_placed": {
 		"event": "block_event",
 		"target": "self",
-		"condition": "query.block_property('bedrock_wiki:block_property') == true"
+		"condition": "query.block_property('bedrock_wiki:block_property') == true" //Optional
 	}
 }
 ```
 
 ### minecraft:on_player_placing
 
+Runs an event when The Player placed the block.
+
 <CodeHeader></CodeHeader>
 
 ```json
 {
 	"minecraft:on_player_placing": {
 		"event": "block_event",
 		"target": "self",
-		"condition": "query.block_property('bedrock_wiki:block_property') == true"
+		"condition": "query.block_property('bedrock_wiki:block_property') == true" //Optional
 	}
 }
 ```
 
 ### minecraft:on_player_destroyed
 
+Runs an event when The Player destroyed the block.
```

Comment:
maybe write "destroys the block", because the other events are written in simple present as well.